src/test/java/com/unstoppabledomains/resolution/ResolutionTest.java

```diff
@@ -256,6 +257,35 @@ public void getOwnerTest() throws NamingServiceException {
         assertEquals("0x003e3cdfeceae96efe007f8196a1b1b1df547eee", owner);
     }
 
+    @Test
+    public void getBatchOwnersTest() throws NamingServiceException {
+        List<String> domains = Arrays.asList("testing.crypto", "unregistered.crypto", "udtestdev-my-new-tls.wallet", "brad.crypto");
+        List<String> owners = resolution.getBatchOwners(domains);
+        String[] correctOwnerAddresses = { "0x58ca45e932a88b2e7d0130712b3aa9fb7c5781e2", null, "0x6ec0deed30605bcd19342f3c30201db263291589", "0x499dd6d875787869670900a2130223d85d4f6aa7"};
```

Comment:
@JohnnyJumper What do you think of we return mapping instead of array? 
```
{'testing.crypto": "0x58ca45e932a88b2e7d0130712b3aa9fb7c5781e2", "unregistered.crypto": null, ...}
```